test/config.js

```diff
@@ -0,0 +1,76 @@
+import path from 'path';
+import test from 'ava';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+
+const fixtureBasePath = path.resolve('test', 'fixtures', 'config');
+
+const callConfigWhenGlobalBinaryUsed = async homedirStub => {
```

Comment:
I'd rename this to `getConfigsWhenGlobalBinaryIsUsed`, and `callConfigWhenLocalBinaryUsed` to `getConfigsWhenLocalBinaryIsUsed`.